sdks/python/apache_beam/testing/analyzers/perf_analysis_utils.py

```diff
@@ -58,8 +55,6 @@ class GitHubIssueMetaData:
 
 def is_change_point_in_valid_window(
     num_runs_in_change_point_window: int, change_point_index: int) -> bool:
-  # If the change point is more than N runs behind the most recent run,
-  # Ignore the change point and don't raise an alert for it.
   return num_runs_in_change_point_window >= change_point_index
```

Comment:
an index typically ranges from 0 to # of entries-1 should it therefore be `return num_runs_in_change_point_window > change_point_index` ?